services/core-web/src/components/common/CoreTooltip.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { PropTypes } from "prop-types";
+import { Tooltip } from "antd";
+import { INFO_CIRCLE } from "@/constants/assets";
+
+const types = {
+  "now-field-origin":
+    "This field is not being sent by NROS or vFCBC. Open the original PDF to see the data.",
+};
+
+const propTypes = {
+  type: PropTypes.oneOf(Object.keys(types)),
+  title: PropTypes.string,
+};
```

Comment:
title looks like its derived from 'props.type' and not a prop that's passed in.
Are there situations where a title prop is passed instead of a type? Can we standardize that?

Reply:
The component will use props.type if one is provided, otherwise it will display props.title. I think it might actually be nicer to restructure this component so that it exports multiple components that all adhere to the same style. E.g., <NowFieldOriginTooltip />, <SomethingElseTooltip />, <FooTooltip />. I think it's nicer to import and use the tooltips you want explicitly rather than having to toss a magic string around (e.g., type="now-field-origin"), which is prone to typos, etc., especially if you're using it a lot across multiple files. I don't expect this minor change to take very long so I'll set it back for ready for review when it's done.